celery/backends/dynamodb.py

```diff
@@ -199,11 +199,11 @@ def _get_or_create_table(self):
             error_code = e.response['Error'].get('Code', 'Unknown')
```

Comment:
Do we need this variable if the condition is removed?
The build is failing because of that.